src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/JintLuceneDocumentConverter.cs

```diff
@@ -207,7 +216,7 @@ private object GetValue(JsValue jsValue)
             }
             else if (jsValue.IsObject())
             {
-                return JavaScriptIndexUtils.StringifyObject(jsValue);
+                return JavaScriptIndexUtils.StringifyObject(jsValue).ToString();
```

Comment:
@talweiss1982 shouldn't we convert this to BlittableObject? Check how they are handled in:
https://github.com/ravendb/ravendb/blob/a0760baf73a63bdbb4a37f0e04f43d46bdd5e8a4/src/Raven.Server/Documents/Indexes/Persistence/Lucene/Documents/LuceneDocumentConverterBase.cs#L318-L349

Reply:
@ppekrol I see your point, but with the current implementation of BlittableObjectReader, we end up creating a lot of arrays, that the memory stream uses, meaning, it's more or less the same as strigify, in my opinion. 
In addition to that, I think that in the buttom line, lucene indexes a string.

Reply:
Then it would be better to pass the ObjectInstance to the converter and handle that there.

Reply:
There = inside LuceneDocumentConverterBase like other types of object.

Reply:
Looking at the problem again, I think that we should do that, although I don't like the fact that we should convert ObjectInstance to blittable to StreamReader to a string just for that, but I don't like having to different approaches performing the same stringification even more :/

Reply:
We can pass `JsObject` and handle that inside `LuceneDocumentConverterBase`. No need for conversion to BlittableObject.

Reply:
I'm not sure I follow.. how will we strigify the jsObject there?

Reply:
we are going to create a BlittableJsonReaderObject here, probably using JsBlittableBridge